src/test/java/org/interledger/codecs/CodecContextFactoryTest.java

```diff
@@ -36,18 +38,21 @@ public void interledger() throws Exception {
     assertTrue(context.hasRegisteredCodec(InterledgerAddress.class));
     assertTrue(context.hasRegisteredCodec(InterledgerPacketType.class));
     assertTrue(context.hasRegisteredCodec(InterledgerPayment.class));
+    assertTrue(context.hasRegisteredCodec(InterledgerPaymentRequest.class));
 
     // TODO: Add ILQP!
-    //    assertTrue(context.readable(InterledgerPacketType.ILQP_QUOTE_LIQUIDITY_REQUEST_TYPE));
+    // assertTrue(context.readable(InterledgerPacketType.ILQP_QUOTE_LIQUIDITY_REQUEST_TYPE));
```

Comment:
Looks like these assertions didn't get enabled when the ILQP code was merged - we should uncomment these lines.